src/components/Dial.js

```diff
@@ -20,7 +20,7 @@ function Dial() {
         }
         window.addEventListener('resize', setView);
         return ()=>window.removeEventListener('resize', setView);
-    })
+    }, []);
```

Comment:
Always remember to pass in a dependency array. An empty array means that the useEffect callback only needs to be called when after the component has initially loaded.

Reply:
I'm a little embarrassed I missed this one.